### DISCUSSION TITLE
"Worms Touched" Item

### DISCUSSION THREAD

**Discussion by FurtadoPires** [ReactiveDrop/reactivedrop_translations#1507] [Q&A] [answered]
From `community\inventory_service\items_.vdf`

I believe this name is a play with "Warm touch" so I'll probably need to adapt to keep it funny. What's exactly the condition to get this item in game?

**Comment by Duke-M-commits** [comment]
Play Operation Sandstorm (a mission of the Reduction campaign) and get thrown in the air by the worm

**Reply by Duke-M-commits** [reply]
What do you mean by wordplay on warm touch?

**Comment by FurtadoPires** [comment]
> What do you mean by wordplay on warm touch?

It's not unusual for developers to put jokes / puns in achievements names, and I think this odd name "Worms Touched" might be a pun with "warm touch", which I think it means "gentle touch" or something similar in english.

> Play Operation Sandstorm (a mission of the Reduction campaign) and get thrown in the air by the worm
Since it related to this, I might be correct since it's a desert (warm) and a giant worm throwing you in the air (touch).

If it's correct, I will need to be creative (again) adapting this to something similar that makes sense in my language 🙃

**Reply by BenLubar** [reply, marked as answer]
This wasn't specifically intended to be a pun, but I'll gladly take credit for the pun if it works

**Reply by FurtadoPires** [reply]
Then It's just me just thinking too much
